docs/user/setup.rst

```diff
@@ -93,16 +93,21 @@ installation of Arch Linux.
     $ sudo dnf groupinstall "Development Tools"
     $ sudo dnf install gc-devel zlib-devel # both optional
 
-**FreeBSD 12.2 and later**
+**FreeBSD 12.4 and later**
 
 .. code-block:: shell
 
-    $ pkg install llvm10
+    $ pkg install llvm # optional
     $ pkg install boehm-gc # optional
 
-*Note:* A version of zlib that is sufficiently recent comes with the
+*Note 1:* Sufficiently recent versions of llvm and zlib come with the
 installation of FreeBSD.
 
+*Note 2:* Only AMD64 and ARM64 architectures are supported.
+
+*Note 3:* Using the boehm GC with multi-threaded binaries doesn't work
+out-of-the-box yet.
+
```

Comment:
Given that a number of Tests, primarily unit-tests, are known to not execute, it may be worthwhile to add a "Note 4" which says something like:
"
A number of Tests, primarily unit-tests, are known to fail
on FreeBSD.  It is believed that the code-under-test is correct and that the defect is in the Test itself.

Work is underway to fix these tests. See [PR](https://github.com/scala-native/scala-native/pull/3625) for a list of tests known to fail at that time."

I/we could create an Issue, with checkboxes, which lists
the known failing tests  (Java 8) and substitute that URL into the text above. 

That way, we are saying what we are doing and doing what
we say.

The last PR in the series which fixes these tests would delete this Note.  I usually dislike creating  work items which have to be deleted later, but this work is in flux.

Your thoughts?